pvlib/bifacial.py

```diff
@@ -62,28 +60,21 @@ def pvfactors_timeseries(
         Back surface reflectivity of PV rows
     horizon_band_angle: float, default 15
         Elevation angle of the sky dome's diffuse horizon band (deg)
-    run_parallel_calculations: bool, default True
-        pvfactors is capable of using multiprocessing. Use this flag to decide
-        to run calculations in parallel (recommended) or not.
-    n_workers_for_parallel_calcs: int, default 2
-        Number of workers to use in the case of parallel calculations. The
-        '-1' value will lead to using a value equal to the number
-        of CPU's on the machine running the model.
 
     Returns
     -------
-    front_poa_irradiance: numeric
+    front_incident_poa_irradiance: numeric
```

Comment:
Are these names tied to pvfactors?  If not, maybe there are names that are more aligned with pvlib patterns, e.g., 
`poa_front ` in place of `front_incident_poa_irradiance`, `poa_front_absorbed` in place of `front_absorbed_poa_irradiance`, or even `effective_irradiance_front` if the spectral adjustment has been built in.  Tossing the idea out.

Reply:
Hi @cwhanse , thanks for the feedback. The names in the docstrings are not tied to anything specific in pvfactors, so I updated it as suggested in f740dea